### DISCUSSION TITLE
Black screen upon login, and when "Trying" elementary OS 7 within the bootable USB.

### DISCUSSION THREAD

**Discussion by irfanwrk** [elementary/triage#263] [Q&A]
What title says, I wanted to upgrade to elementary OS 7 (6 worked perfect for me, don't know why this would be happening for 7). I booted off the drive, quick and easy, did my partitioning, and it installed perfectly fine.

I boot, I login and.... nothing.

A black screen, a slightly rounded red box that shows up for one second and then absolutely nothing.

I'm like, what? Did I install it wrong?
Tried a bunch of times, same occurrance.

So I tried actually "trying" it from the pen drive using the Try elementary OS 7 option within the installer and I got the same results.

Any ideas, anyone? I really wanna use elementary OS 7 and I have no idea what to do.

EDIT: My laptop is an ASUS X541SA, with 4 gigabytes of ram and an Intel Pentium N3710. I don't know what other information could be required, but please ask if need be!

EDIT: So I tried installing eOS 7 once more and uh, not it does not let me create my user, immediately after boot it puts me into a black screen. My bad guys.

**Comment by lenemter** [comment]
Were you able to create your user after installing?

**Reply by irfanwrk** [reply]
I was, yes. As I said, I had made my account, and then was prompted to login to the DE. Once I logged in, the above issues were presented to me.

**Reply by lenemter** [reply]
That's weird. What is your GPU?

**Reply by irfanwrk** [reply]
I don't have one, it's an integrated GPU that came with the processor, since this is just a laptop. I want to emphasize that 6 worked perfectly fine (and rather smoothly compared to its less elegant counterparts, such as Fedora or pure Ubuntu).

**Comment by lenemter** [comment]
Oh, in this case maybe updating the systm might help. Press Ctrl + Alt + F1, type your username and password and type

```
sudo apt update
sudo apt upgrade -y
Reboot
```

**Reply by lenemter** [reply]
Oops replied in the wrong place

**Reply by irfanwrk** [reply]
No worries!

**Reply by irfanwrk** [reply]
Also, how may I do this? Like exactly when and where should I try this during the whole process?

**Comment by irfanwrk** [comment]
@lenemter I cannot create a user when I install eOS, my bad. What do you suggest?

**Comment by irfanwrk** [comment]
UPDATE: So after messing around a bit, I eventually went to tty1, and while pressing a bunch of the function keys I pressed the F7 key to be thrust into the account creation screen.

I created my account, logged in and went to the desktop environment but everything was quite laggy. Restarted, it brought me to the login interface, except it one again went black with only my initials displayed in red. I discovered I could type my password in, since my initials disappeared once I entered the password and pressed enter.

I then discovered _nomodeset_ which I put into my GRUB settings. It made the black screen disappear but, the OS was still laggy. I realized I couldn’t change the brightness, so I deduced that the OS wasn’t detecting the integrated GPU on my Intel Pentium.

Any suggestions? I can use nomodeset to boot into the environment but, I can’t do sudo apt update. I truly want to use this OS and have been trying to fix it to that end, but I don’t know what is going on. Doing apt update and then apt full-upgrade did not help the situation at all.

**Reply by lenemter** [reply]
What is your CPU? Maybe shipped Linux kernel or Mesa doesn't properly support it

**Reply by irfanwrk** [reply]
> What is your CPU? Maybe shipped Linux kernel or Mesa doesn't properly support it

Intel Pentium N3710

**Reply by lenemter** [reply]
That's weird. I don't know why this is happening, sorry.

**Reply by irfanwrk** [reply]
> That's weird. I don't know why this is happening, sorry.

It’s fine, thanks for your help!
